doc/usecases/polyculture/review_plant_relationships.md

```diff
@@ -0,0 +1,28 @@
+# Use Case: Review Plant Relationships
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User, App User
+- **Brief:** The quality of a user created plant relationship is assessed by others.
+- **Status:** Assigned
+- **Assignee:** Benjamin
+
+## Scenarios
+
+- **Precondition:**
+  - The user is logged into the app.
+  - She is looking at a [plant relationship created by another user](add_plant_relationships.md).
+- **Main success scenario:**
+  - The user comments on the relationship to note something that seems relevant to her.
+  - The user approves the relationship because she agrees with it.
+- **Alternative scenario:**
+  - The user only comments on the relationship but doesn't approve it and might suggest a different relationship (e.g. neutral). 
+  - No changes are made to the relationships confidence score.
+  - One administrator decides on the outcome.
+- **Error scenario:**
+- **Postcondition:**
+  - The number of approvals are counted for the administrator to decide.
+    Relationships with higher confidence score will have a higher weight when [suggesting plants](suggest_plants.md) or [suggesting alternatives](suggest_alternatives.md).
+- **Non-functional Constraints:**
```

Comment:
You have no Non-funcational Contstraints, here are some suggestions:
 - The review interface should be intuitive and easy to use, allowing users to quickly assess and comment on plant relationships.
 - The system should ensure that users can only review and comment on plant relationships they have access to, and that their input is properly validated. (Data Integrity)
 - The system should prevent security vulnerabilities, such as cross-site scripting (XSS) attacks, by properly sanitizing and validating the reviews before displaying it on the app.

Reply:
Regarding your points.

1. I don't think this needs to be noted as it would have to be added for each use case.
See https://github.com/ElektraInitiative/PermaplanT#non-functional-goals
2. I'm not sure if there are any restrictions. I assumed that global relationship proposals are public.
Is this correct @markus2330 ?
3. Again this is one of those requirements that would be relevant for almost each use case.

Reply:
1. You're absolutely right, I should have payed more attention
3. We should then add Security to the non-functional-goals I suppose, what do you think @markus2330 ?

Reply:
> I'm not sure if there are any restrictions. I assumed that global relationship proposals are public.
Is this correct @markus2330 ?

Yes, public.

> We should then add Security to the non-functional-goals I suppose, what do you think @markus2330 ?

Only for login etc.